### PR TITLE
copy project files

### PR DESCRIPTION
**Pull request by mercihabam**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            <div class="flex gap-8 flex-wrap justify-center sm:justify-start">
+                <div class="footer--link">About</div>
+                <div class="footer--link">Advertising</div>
+                <div class="footer--link">Business</div>
+                <div class="footer--link">How Search works</div>
+            </div>
+            <div class="flex gap-8 flex-wrap justify-center sm:justify-start">
+                <div class="footer--link">Privacy</div>
+                <div class="footer--link">Terms</div>
+                <div class="footer--link">Settings</div>
```
Comment (mutesialine) [inline index.html]: according to the instructions use tailwindcss in your index.html remove those classes

File: style.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+@layer components {
+    .action-btn{
+        @apply outline-none bg-gray-50 border border-transparent text-sm text-gray-800 px-4 py-2 hover:border-gray-300 hover:shadow
+    }
+    .lang-link{
+        @apply text-blue-800 hover:underline
+    }
+    .footer--link{
+        @apply text-sm text-gray-500 cursor-pointer hover:underline
+    }
```
Comment (mutesialine) [inline style.css]: these classes in style .css  are not allowed style it in index.html

File: index.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Google</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css">
```
Comment (mutesialine) [inline index.html:9]: no framework is allowed to use  try to inspect on  google site  and find icon they used or visit heroicon it has icon made by tailwindcss

File: index.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        </div>
+        <div class="flex justify-center mt-6 gap-5 relative">
+            <button class="action-btn">Google Search</button>
+            <button class="action-btn">I'm Feeling Lucky</button>
+        </div>
+        <div class="flex justify-center-items-center gap-4 text-xs mt-6">
+            <div class="text-gray-700">Google offered in:</div>
+            <a href="#" class="lang-link">Français</a>
+            <a href="#" class="lang-link">Kiswahili</a>
+            <a href="#" class="lang-link">Kinyarwanda</a>
```
Comment (mutesialine) [inline index.html]: also here remove classes   use tailwindcss in html

**Review by mutesialine — COMMENTED** [review]
well done @jacob-js  i left some comment below try to make some changes on few things

File: index.html [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+        <div class="cursor-pointer text-sm font-light hover:underline">Gmail</div>
+        <div class="cursor-pointer text-sm font-light hover:underline">Images</div>
+        <div class="cursor-pointer w-9 h-9 rounded-full hover:bg-gray-200 flex justify-center items-center">
+            <i class="bi bi-grid-3x3-gap-fill"></i>
+        </div>
+        <button class="bg-blue-600 text-white outline-none px-4 py-2 rounded-md hover:shadow-md font-medium">Sign in</button>
+    </nav>
+    <main class="flex flex-col items-center min-h-[80vh]">
+        <img src="./img/logo.png" alt="" class="w-52 lg:w-68">
+        <div class="border border-gray-300 rounded-3xl relative flex items-center px-4 mt-10 w-[90%] lg:w-2/5 hover:shadow-md">
```
Comment (serapieTuyishime) [inline index.html]: this 90% should be defined in tailwind.config.js

File: index.html [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+            <i class="bi bi-search text-gray-400 text-sm"></i>
+            <input type="text" class="outline-none bg-none border-none py-[10px] ml-3 w-full">
```
Comment (serapieTuyishime) [inline index.html]: no abstract classes

File: index.html [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+            <img src="./img/mic.svg" alt="" class="w-5">
+        </div>
+        <div class="flex justify-center mt-6 gap-5 relative">
+            <button class="action-btn">Google Search</button>
```
Comment (serapieTuyishime) [inline index.html]: no BEN should be used, all the tailwindcss classes should be put here.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+            <button class="action-btn">I'm Feeling Lucky</button>
+        </div>
+        <div class="flex justify-center-items-center gap-4 text-xs mt-6">
+            <div class="text-gray-700">Google offered in:</div>
+            <a href="#" class="lang-link">Français</a>
```
Comment (serapieTuyishime) [inline index.html]: no BEM should be used.

File: index.html [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+            <a href="#" class="lang-link">Kiswahili</a>
+            <a href="#" class="lang-link">Kinyarwanda</a>
+        </div>
+    </main>
+
+    <footer class="bg-gray-100 w-full fixed bottom-0">
+        <div class="text-gray-500 px-10 py-3 text-sm">Your Country</div>
+        <div class="border w-full"></div>
+        <div class="footer--links flex justify-center flex-wrap gap-5 sm:justify-between px-10 py-3">
```
Comment (serapieTuyishime) [inline index.html]: footer--links should be replaced by its corresponding classes

File: index.html (hunk 1 above)
Comment (serapieTuyishime) [inline index.html]: even though it is repetition, replace all classes in footer--link with their classes

**Review by serapieTuyishime — CHANGES_REQUESTED** [review]
remove the BEM and abstract classes

File: index.html (hunk 6 above)
Comment (serapieTuyishime) [reply index.html]: py-[10px] is an abstract class and it is not allowed you should do this in config file

**Review by serapieTuyishime — CHANGES_REQUESTED** [review]

**Review by serapieTuyishime — APPROVED** [review]
Thanks for making the changes

**Review by mutesialine — APPROVED** [review]

**Review by karemarsy — APPROVED** [review]

**Pull request merged**